tests.sh

```diff
@@ -220,6 +220,54 @@ build_python() {
   cd ..
 }
 
+build_python27() {
```

Comment:
most of these functions are the same, can we call a helper function and just pass in the envlist as an argument?

Reply:
done